bin/i18n/sync-codeorg-in.rb

```diff
@@ -196,14 +160,72 @@ def localize_block_content
     blocks[name]['options'] = args_with_options unless args_with_options.empty?
   end
 
-  copy_to_yml('blocks', blocks)
+  File.open("dashboard/config/locales/blocks.en.yml", "w+") do |f|
+    f.write(to_crowdin_yaml({"en" => {"data" => {"blocks" => blocks}}}))
+  end
+end
+
+def select_redactable(i18n_strings)
+  redactable_content = %w(
+    authored_hints
+    long_instructions
+    short_instructions
+  )
+
+  redactable = i18n_strings.select do |key, _|
+    redactable_content.include? key
+  end
+
+  if i18n_strings.key? "contained levels"
+    contained_levels = i18n_strings["contained levels"].map do |contained_level|
+      select_redactable(contained_level)
+    end
+    contained_levels.select! do |result|
+      !result.blank?
```

Comment:
nit: use `result.present?` instead of `!result.blank?` 

(also, [this](https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html) is my favorite article explaining these types of ActiveSupport helpers -- .nil?, .empty?, .blank?, etc -- and i refer to it almost every day)